src/core/Property.h

```diff
@@ -0,0 +1,211 @@
+// Copyright © 2008-2021 Pioneer Developers. See AUTHORS.txt for details
```

Comment:
In the year __2022__, _(people drink soylent green... seriously, the movie takes place in 2022)_